Remotion/Web/UnitTests/Core/WebStringTest.cs

```diff
@@ -255,6 +255,22 @@ public void ToString_WithPlainTextTextAndHtmlWithEncodedLineBreaksEncoding_Encod
       Assert.That (WebString.CreateFromText ("aoe   \" & ' < > é \r \n \r\n").ToString (WebStringEncoding.HtmlWithTransformedLineBreaks), Is.EqualTo ("aoe &#160; &quot; &amp; &#39; &lt; &gt; &#233; <br /> <br /> <br />"));
     }
 
+    [Test]
+    public void ToPlainTextString_WithPlainTextText_Succeeds ()
+    {
+      var plainTextString = WebString.CreateFromText ("test").ToPlainTextString();
+
+      Assert.That (plainTextString.GetValue(), Is.EqualTo ("test"));
+    }
+
+    [Test]
+    public void ToPlainTextString_WithEncodedText_ThrowsInvalidOperationException ()
+    {
+      Assert.That(
+          () => WebString.CreateFromHtml ("test").ToPlainTextString(),
+          Throws.TypeOf<InvalidOperationException>());
```

Comment:
Please assert message too. Helps with making sure that a) the correct exception case gets hit and b) the message works (for constructed messages)